Makefile

```diff
@@ -12,6 +12,10 @@ include build/setup.mk
 
 BUNDLE_NAME ?= $(PLUGIN_ID)-$(PLUGIN_VERSION).tar.gz
 
+ifneq ($(wildcard build/custom.mk),)
+	include build/custom.mk
```

Comment:
@hanzei What type of custom commands would be included in this makefile? I see that each plugin has their own build/setup.mk that is already included in Makefile. Could these custom commands be added there ? 🤔